docs/using-the-jdbc-driver/DataSource.md

```diff
@@ -5,9 +5,9 @@ You can use the `DriverManager` class or a datasource to establish a new connect
 
 To establish a connection with the AwsWrapperDataSource, you must:
 
-1. Configure the property names for the underlying driver-specific datasource. 
-2. Target a driver-specific datasource.
-3. Configure the driver-specific datasource.
+1. Target a driver-specific datasource.
```

Comment:
Could this target be select? We use the word target a lot with the underlying target driver, so maybe we should not use it here

I think it could be something like "Select a datasource specific to the driver you want to use" or similar